include/quic/endpoint.hpp

```diff
@@ -109,27 +188,10 @@ namespace oxen::quic
         // Accepts new connection, returning either a ptr to the Connection
         // object or nullptr if error. Virtual function returns nothing --
         // overrided by Client and Server classes
-        virtual Connection* accept_initial_connection(Packet& pkt, ConnectionID& dcid) = 0;
+        Connection* accept_initial_connection(Packet& pkt, ConnectionID& dcid);
 
-        // NOTE: this may not be necessary for a generalizable quic library,
-        //      as it is a lokinet-specific implementation. However, it may be
-        //      useful in the future to be able to add our own headers to quic
-        //      packets for whatever purpose
-        //
-        // Writes packet header to the beginning of this.buf; this header is
-        // prepended to quic packets to handle quic server routing, consists of:
-        // - type [1 byte]: 1 for client->server packets; 2 for server->client packets
-        //      (other values reserved)
-        // - port [2 bytes, network order]: client pseudoport (i.e. either a source or
-        //      destination port depending on type)
-        // - ecn value [1 byte]: provided by ngtcp2 (Only the lower 2 bits are actually used).
-        //
-        // \param psuedo_port - the remote's pseudo-port (will be 0 if the remote is a
-        //      server, > 0 for a client remote)
-        // \param ecn - the ecn value from ngtcp2
-        //
-        // Returns the number of bytes written to buf
-        virtual size_t write_packet_header(uint16_t pseudo_port, uint8_t ecn) { return 0; }
+	  private:
+	  	//
```

Comment:
a comment nobody can read it perhaps a bit *too* private...

Reply:
its our little secret tom